ci/eval/compare/default.nix

```
@@ -71,7 +71,9 @@ let
     getLabels
     ;
 
-  getAttrs = dir: builtins.fromJSON (builtins.readFile "${dir}/outpaths.json");
+  getAttrs =
+    dir:
+    builtins.fromJSON (builtins.unsafeDiscardStringContext (builtins.readFile "${dir}/outpaths.json"));
```

```suggestion
  getAttrs =
    dir:
    let
      raw = builtins.readFile "${dir}/outpaths.json";
      # The file contains Nix paths; we need to ignore them for evaluation purposes,
      # else there will be a "is not allowed to refer to a store path" error.
      data = builtins.unsafeDiscardStringContext raw;
    in
    builtins.fromJSON data;
```

I think it's worth adding a small note about this and why it's here.